include/seqan/simd.h

```diff
@@ -60,6 +60,16 @@
     #undef SEQAN_UMESIMD_ENABLED
 #endif
 
+// SIMD operations make only sense on modern 64bit architectures.
+#if SEQAN_IS_32_BIT
+    #if !(defined(NDEBUG) || defined(SEQAN_ENABLE_TESTING))
+    #pragma message("SIMD acceleration is only available on 64bit systems")
```

Comment:
That is only to prevent ```#pragma message``` ? 
Can you then please add a TODO, that on jenkins we remove the restriction and filter the warning messages out after the run.